tests/installation/bootloader_s390.pm

```diff
@@ -301,4 +302,25 @@ sub run {
     $self->result('ok');
 }
 
+sub post_fail_hook {
+    my $s3270 = console('x3270');
+    my $r;
+
+    if (check_screen 'linuxrc') {
```

Comment:
meh, as you probably know by know I consider a check_screen with default timeout a bad practice but this time, being in the `post_fail_hook` … ok